defaults/preferences/zotero.js

```diff
@@ -186,6 +186,9 @@ pref("extensions.zotero.connector.repo.lastCheck.repoTime", 0);
 pref("extensions.zotero.fallbackLauncher.unix", "/usr/bin/xdg-open");
 pref("extensions.zotero.fallbackLauncher.windows", "");
 
+// Custom pdf reader
+pref("extensions.zotero.pdf.launcher", "");
```

Comment:
While we probably won't bother doing this for anything else, it's not impossible, so how about `extensions.zotero.fileHandler.pdf` and generalize the comment to "File handlers"? Also move above the `fallbackLauncher` prefs.

Reply:
You're correct. That sounds better.